adala/runtimes/_litellm.py

```diff
@@ -33,6 +39,25 @@
 logger = logging.getLogger(__name__)
 
 
+# basically only retrying on timeout, incomplete output, or rate limit
+# https://docs.litellm.ai/docs/exception_mapping#custom-mapping-list
+# NOTE: token usage is only correctly calculated if we only use instructor retries, not litellm retries
+# https://github.com/jxnl/instructor/pull/763
+retry_policy = dict(
```

Comment:
nit: can we make this in all caps to make it clear it's top-level scope? (I know we haven't been consistent about this in general, but I was looking around for it as a param and I think it'd help readability a bit)